src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/AbstractExpressionHandler.java

```diff
@@ -251,26 +251,32 @@ protected boolean shouldIncreaseIndent() {
     /**
      * Check the indentation for a set of lines.
      *
-     * @param lines              the set of lines to check
+     * @param astSet             the set of abstract syntax tree to check
      * @param indentLevel        the indentation level
      * @param firstLineMatches   whether or not the first line has to match
      * @param firstLine          first line of whole expression
+     * @param allowNesting       whether or not subtree nesting is allowed
      */
-    private void checkLinesIndent(LineSet lines,
+    private void checkLinesIndent(DetailAstSet astSet,
                                   IndentLevel indentLevel,
                                   boolean firstLineMatches,
-                                  int firstLine) {
-        if (!lines.isEmpty()) {
+                                  int firstLine,
+                                  boolean allowNesting) {
+        if (!astSet.isEmpty()) {
             // check first line
-            final int startLine = lines.firstLine();
-            final int endLine = lines.lastLine();
-            final int startCol = lines.firstLineCol();
+            final DetailAST startLine = astSet.firstLine();
+            final int endLine = astSet.lastLine();
```

Comment:
Naming is confusing. They are both called "Line" but one is an int and the other is an AST.
`startLine` => `startLineAst`.

Reply:
Done